packages/components/datetime/src/RelativeDateTime.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import dayjs from 'dayjs';
+import utcPlugin from 'dayjs/plugin/utc';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import calendarPlugin from 'dayjs/plugin/calendar';
+import isTodayPlugin from 'dayjs/plugin/isToday';
+dayjs.extend(utcPlugin);
+dayjs.extend(relativeTime);
+dayjs.extend(calendarPlugin);
+dayjs.extend(isTodayPlugin);
+
+export interface RelativeDateTimeProps {
```

Comment:
All comments for `DataTime` are relevant to this file too.

Reply:
Do you mean polymorphic and common props?